scripts/algorithms/l2-svm-predict.dml

```diff
@@ -25,6 +25,19 @@
 # Given ground truth labels, the script will compute an 
 # accuracy (%) for the predictions
 #
+# INPUT PARAMETERS:
+# ---------------------------------------------------------------------------------------------
+# NAME      TYPE        DEFAULT     MEANING
+# ---------------------------------------------------------------------------------------------
+# X         String      ---         Location to read the test feature matrix X
+# Y         String      ---         Location to read the true label matrix Y
```

Comment:
Unfortunately, this script is a bit of a mess in that it combines "predictions" (thus the name of the file), along with evaluation (accuracy, etc.).  At some point, we'll redo all of these scripts in a coherent manner.  For now though, can you note that `Y` is optional, and only needed if the user wishes to evaluate the performance (accuracy, confusion) of the model?